packages/grafana-toolkit/src/cli/tasks/searchTestDataSetup.ts

```diff
@@ -55,7 +55,7 @@ export async function setDashboardAcl(dashboardId: any, aclList: any) {
   await client.post(`/dashboards/id/${dashboardId}/permissions`, { items: aclList });
 }
 
-const searchTestDataSetupRunnner: TaskRunner<SearchTestDataSetupOptions> = async ({ count }) => {
+const searchTestDataSetupRunner: TaskRunner<SearchTestDataSetupOptions> = async ({ count }) => {
```

Comment:
Afaiu, this is a local -- yes, it's exposed below, but only the object, not the name